hosts/galileo/ssi/__init__.py

```diff
@@ -0,0 +1,298 @@
+################################################################################
+# hosts/galileo/ssi/__init__.py
+################################################################################
+import numpy as np
+import julian
+import cspyce
+import vicar
+import pdstable
+import pdsparser
+import oops
+
+from hosts         import pds3
+from hosts.galileo import Galileo
+
+################################################################################
+# Standard class methods
+################################################################################
+def from_file(filespec,
+              return_all_planets=False, full_fov=False, **parameters):
+    """A general, static method to return a Snapshot object based on a given
+    Galileo SSI image file.  By default, only the valid image region is
+    returned.
+
+    Inputs:
+        full_fov:           If True, the full image is returned with a mask
```

Comment:
filespec is also an input (yes I know it's missing from Cassini ISS). Order of other two comments should be swapped.

Reply:
corrected in cassini, juno, and voyager as well